jkube-kit/generator/java-exec/src/test/java/org/eclipse/jkube/generator/javaexec/JavaExecGeneratorMainClassDeterminationTest.java

```diff
@@ -90,22 +93,21 @@ public void testMainClassDeterminationFromConfig() {
 
         List<ImageConfiguration> customized = generator.customize(new ArrayList<>(), false);
 
-        assertEquals("1 images returned", 1, customized.size());
-
-        ImageConfiguration imageConfig = customized.get(0);
-
-        assertEquals("Image name", "TheImageName", imageConfig.getName());
-        assertEquals("Main Class set as environment variable",
-                "the.main.ClassName",
-                imageConfig.getBuildConfiguration().getEnv().get(JavaExecGenerator.JAVA_MAIN_CLASS_ENV_VAR));
+        assertThat(customized).singleElement()
+                        .hasFieldOrPropertyWithValue("name", "TheImageName")
+                        .extracting(ImageConfiguration::getBuildConfiguration)
+                        .extracting(BuildConfiguration::getEnv)
+                        .asInstanceOf(InstanceOfAssertFactories.MAP)
+                        .as("Main Class set as environment variable")
+                        .containsEntry(JavaExecGenerator.JAVA_MAIN_CLASS_ENV_VAR, "the.main.ClassName");
```

Comment:
Could we use literals here instead of referencing the `JavaExecGenerator` constants?